.github/workflows/build-pr.yml

```diff
@@ -82,8 +82,9 @@ jobs:
 
       - name: Run smoke test against OSS image
         timeout-minutes: 2
-        run: |
-          .github/scripts/simple-smoke-test.sh hazelcast-oss:test ${{ env.test_container_name_oss }} oss ${{ needs.prepare.outputs.HZ_VERSION_OSS }}
+        run: |          
+          . .github/scripts/docker.functions.sh
+          .github/scripts/simple-smoke-test.sh hazelcast-oss:test ${{ env.test_container_name_oss }} oss ${{ needs.prepare.outputs.HZ_VERSION_OSS }} "$(get_default_jdk hazelcast-oss)"
```

Comment:
We don't deploy OSS for PATCH
Just checking this is accounted for?

Reply:
> We don't deploy OSS for PATCH Just checking this is accounted for?

This isn’t a release - in the PR, we build the images and push them to a (local) registry - for both OS & EE, always.

Reply:
understood.